### PR TITLE
[libkeyfinder] add new port with libkeyfinder 2.2.4

### PR DESCRIPTION
**Pull request by Be-ing**
**Describe the pull request**

- Which triplets are supported/not supported? Have you updated the CI baseline?
All platforms are supported. It's just a small C++ library that only depends on FFTW.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
AFAIK yes

### CODE REVIEW

File: ports/libkeyfinder/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "libkeyfinder",
+  "version-string": "2.2.4",
+  "description": "Musical key detection for digital audio",
+  "homepage": "https://github.com/mixxxdj/libkeyfinder",
+  "dependencies": [
+    "catch2",
```
Comment (Be-ing) [inline ports/libkeyfinder/vcpkg.json:7]: This dependency could be removed if BUILD_TESTING=OFF is passed to cmake. Is there a policy on this?

Comment (Be-ing) [reply ports/libkeyfinder/vcpkg.json:7]: oh yeah, I'll change this https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#do-not-build-testsdocsexamples-by-default

File: ports/libkeyfinder/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+)
+
+vcpkg_install_cmake()
+vcpkg_copy_pdbs()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/KeyFinder TARGET_PATH share/KeyFinder)
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+# Handle copyright
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libkeyfinder RENAME copyright)
```
Comment (NancyLi1013) [inline ports/libkeyfinder/portfile.cmake:25]:
```suggestion
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @Be-ing 

Thanks for your PR.

Have you tested the feature?

**Comment by Be-ing** [comment]
Yes I have tested it here: https://github.com/mixxxdj/mixxx/pull/3594

**Review by NancyLi1013 — APPROVED** [review]

**Comment by Be-ing** [comment]
mixxx-windows
Thanks to vcpkg we've been able to add several libraries to our Windows builds (libkeyfinder, lilv, libmodplug) that were too much of hassle to add with our old system of custom batch scripts. :grinning: Just in time for [PawPaw](https://kx.studio/News/?action=view&url=announcing-pawpaw-cross-platform-lv2-ports-for-macos-and-windows) to make LV2 audio plugins available on Windows!

**Comment by Be-ing** [comment]
What is left before this can be merged?

**Comment by NancyLi1013** [comment]
For me, nothing needs to be done here.

**Comment by Be-ing** [comment]
> Thanks to vcpkg we've been able to add several libraries to our Windows builds (libkeyfinder, lilv, libmodplug) that were too much of hassle to add with our old system of custom batch scripts. grinning Just in time for PawPaw to make LV2 audio plugins available on Windows!

I wrote a [blog post](https://mixxx.org/news/2021-02-23-build-infrastructure-updates/) detailing this if you're curious.

**Pull request merged**